03-Binary-Search-Trees/01-Binary-Search-Trees.md

```diff
@@ -1,5 +1,6 @@
 # Binary Search Trees
 
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=a2007d27-80eb-4d19-a549-afcc004a1759&autoplay=false&offerviewer=true&showtitle=true&showbrand=true&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
```

Comment:
One thing that led to a little bit of confusion on my part was the naming of the videos - I believe two of the videos share the same title "Consider a Non-Linear Data Structure". I'm not sure if this was intentional but I might recommend changing the title of the second video to "Binary Search Tree Design" or "Binary Search Tree Data Structure" or something similar.

Reply:
Nice catch, thanks!